sdk/keyvault/azure-security-keyvault-keys/README.md

```diff
@@ -15,7 +15,7 @@ Maven dependency for Azure Key Client library. Add it to your project's pom file
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-security-keyvault-keys</artifactId>
-    <version>4.0.0</version>
+    <version>4.0.0-preview.5</version>
```

Comment:
This is undo'ing some of the changes that @hemanttanwar did in PR https://github.com/Azure/azure-sdk-for-java/pull/5991. I know @hemanttanwar and @JimSuplizio were going to discuss what to do here.

Reply:
so long as we get an ACK from @hemanttanwar then we're OK.

Temporarily regressing the README.md here is fine, as we'll need to re-run this script again (correcting this line) as part of updating keyvault keys

Reply:
OK - are we good to go then? Just waiting for a CI pass (minor readme tweak) and then this should be good to merge.